packages/package-builder/src/Reflection/PrivatesCaller.php

```diff
@@ -17,7 +17,7 @@ final class PrivatesCaller
      * @param object|string $object
      * @param mixed[] $arguments
      */
-    public function callPrivateMethod($object, string $methodName, ...$arguments)
+    public function callPrivateMethod($object, string $methodName, array $arguments)
```

Comment:
This is a breaking change. Symplify was released as `v9.0.26` immediately, but Rector was not, and it references this code, so it's now failing in production.

Right now, doing `composer update` installs Symplify `9.0.26` and Rector version `0.9.5`, and doing `rector process ...` fails (so my CI is broken):

```
PHP Fatal error:  Uncaught TypeError: Argument 3 passed to Symplify\PackageBuilder\Reflection\PrivatesCaller::callPrivateMethod() must be of the type array, object given, called in /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/rector/rector/src/Console/Style/SymfonyStyleFactory.php on line 32 and defined in /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/symplify/package-builder/src/Reflection/PrivatesCaller.php:20
Stack trace:
#0 /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/rector/rector/src/Console/Style/SymfonyStyleFactory.php(32): Symplify\PackageBuilder\Reflection\PrivatesCaller->callPrivateMethod(Object(Symfony\Component\Console\Application), 'configureIO', Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/rector/rector/bin/rector.php(39): Rector\Core\Console\Style\SymfonyStyleFactory->create()
#2 /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/rector/rector/bin/rector(4): re in /Users/mac/GitRepos/GitHub/Projects/leoloso/PoP/vendor/symplify/package-builder/src/Reflection/PrivatesCaller.php on line 20
```

Reply:
Releasing Symplify with breaking changes may break Rector in production, so what about hosting both projects in the same monorepo? 🤔

Reply:
Without mixing their packages:

- `projects/Symplify/packages/...`
- `projects/Rector/packages/...`

Reply:
I'm aware of it.
Rector release fixes it, see: https://github.com/rectorphp/rector/issues/5124